build/README.md

```diff
@@ -234,6 +236,41 @@ $ make minikube-transfer-image TAG=myimage:0.1
 
 Running end-to-end tests on minikube is done via the `make minikube-test-e2e` target. This target use the same `make test-e2e` but also setup some prerequisites for use with a minikube cluster.
 
+### Running a Custom Test Environment
+
+This section is addressed to developers using a custom Kubernetes provider, a custom image repository and/or multiple test clusters.
+
+Prerequisites:
+- a(some) running k8s cluster(s)
+- Have kubeconfig file(s) ready
+- docker must be logged into the image repository you're going to use
+
+To begin, you need to set up the following environment variables:
+- `KUBEPATH` should point to the folder containing your kubeconfig file; 
```

Comment:
I feel like `KUBEPATH` and `KUBECONFIGFILE` could be replaced by `KUBECONFIG` as a full path. WDYT?

Reply:
I agree. @Kuqd WDYT?

Reply:
yes good call plus both ` KUBEPATH` and `KUBECONFIGFILE` are not standard, while `KUBECONFIG` is , so there is a greater chance user have this set.

Reply:
agreed.